source/gems/dry-validation/basics/macros.html.md

```diff
@@ -9,7 +9,9 @@ This document describes available built-in macros.
 
 ### filled
 
-Use it when a value is expected to be filled.
+Use it when a value is expected to be filled. "filled" means that the value is
+non-nil, and if the value is a `String`, `Hash`, or `Array` then `value.empty?`
+returns `false`.
```

Comment:
Could you remove line hard-wrapping as it messes up formatting when middleman builds it?

Apart from that, I would probably just write "In case of `String`, `Hash` and `Array` their `empty?` method is used". WDYT?